src/main/java/uk/gov/companieshouse/insolvency/data/api/InsolvencyApiService.java

```diff
@@ -15,12 +15,15 @@
 import uk.gov.companieshouse.insolvency.data.exceptions.MethodNotAllowedException;
 import uk.gov.companieshouse.insolvency.data.exceptions.ServiceUnavailableException;
 import uk.gov.companieshouse.insolvency.data.model.InsolvencyDocument;
+import uk.gov.companieshouse.insolvency.data.util.DateTimeFormatter;
 import uk.gov.companieshouse.logging.Logger;
 
 @Service
 public class InsolvencyApiService {
 
     private static final String CHANGED_RESOURCE_URI = "/private/resource-changed";
+    private static final String DATE_TIME_FORMAT_SECOND_PRECISION = "yyyy-MM-dd'T'HH:mm:ss";
```

Comment:
unneeded variable here.

Reply:
As the value of the String isn't easy to read/understand what it's for, the constant is more readable (e.g., it's a date time format with second precision).

I think it's generally nice to have constants like this if the value of the string isn't particularly easy to understand.

Reply:
I see what you mean. It was meant to go in the DateTimeFormatter class.